ott/core/quad_problems.py

```diff
@@ -121,6 +121,14 @@ class QuadraticProblem:
     gw_unbalanced_correction: True (default) if the unbalanced version of
       Sejourne et al. (Neurips 2021) is used, False if tau_a and tau_b
       only affect the inner Sinhkorn loop.
+    ranks: Ranks of the cost matrices, see
+      :meth:`~ott.geometry.geometry.Geometry.to_LRCGeometry`. Used when
+      geometries are *not* :class:`~ott.geometry.pointcloud.PointCloud` with
+      `'sqeucl'` cost function. If `-1`, the geometries will not be converted
+      to low-rank. If :class:`int`, rank shared across all geometries.
```

Comment:
If :class:`int`, geometries provided as inputs will be converted to low-rank geometries using ...

maybe discuss the case where we have a Tuple and the order with which it is used in that case?

Reply:
Done!